tests/steps/given.ts

```diff
@@ -4,6 +4,7 @@ import { Given, Then } from "@cucumber/cucumber";
 import GPS from '@/models/GPS';
 import ConnectMongoDB from '@/lib/mongodb';
 import AISShips from '@/models/AISShips';
+import GlobalPath from '@/models/GlobalPath';
 
 Given('I clear the database', async function () {
     const db = await ConnectMongoDB();
```

Comment:
Please add a line below to clear all the globalpath data (following what GPS and AISShips do)

Reply:
I added `await GlobalPath.deleteMany();` underneath, but some GlobalPath data is still showing up when going to the endpoint - am I missing something else ? picture of data is attached below: 

![image](https://github.com/UBCSailbot/website/assets/92189215/3e1b61c7-9543-4e19-b1ef-de3ae7e1ed8f)

Reply:
Can you run it a couple of times consecutively? If it only returns one document with the same three waypoints, then it's working. 

If you didn't have the `await GlobalPath.deleteMany()`, then if you were to run the test multiple times, the documents would stack up.